extension/json/test/scan_json.test

```diff
@@ -104,23 +104,45 @@ Cannot read from JSON Object to FLOAT
 {a: 1, b: 2024-02-11}
 {a: , b: 2000-01-01}
 
--CASE CopyFromJsonIgnoreErrors
+-CASE CopyFromJsonIgnorePKErrorsArrayFormat
 -STATEMENT LOAD EXTENSION "${KUZU_ROOT_DIRECTORY}/extension/json/build/libjson.kuzu_extension";
 ---- ok
 -STATEMENT CREATE NODE TABLE person(ID UINT64, PRIMARY KEY (ID))
 ---- ok
 -STATEMENT COPY person FROM "${KUZU_ROOT_DIRECTORY}/dataset/json-error/pk_constraint_violation.json" (IGNORE_ERRORS=true);
 ---- ok
--STATEMENT CALL show_warnings() RETURN message
+-STATEMENT CALL show_warnings() RETURN message, file_path, line_or_record_number
 ---- 2
-Found NULL, which violates the non-null constraint of the primary key column.
-Found duplicated primary key value 2, which violates the uniqueness constraint of the primary key column.
+Found NULL, which violates the non-null constraint of the primary key column.|${KUZU_ROOT_DIRECTORY}/dataset/json-error/pk_constraint_violation.json|3
+Found duplicated primary key value 2, which violates the uniqueness constraint of the primary key column.|${KUZU_ROOT_DIRECTORY}/dataset/json-error/pk_constraint_violation.json|5
 -STATEMENT MATCH (p:person) RETURN p.ID
 ---- 3
 1
 2
 3
 
+-CASE CopyFromJsonIgnoreIndexLookupErrorsNewlineDelimitedFormat
+-STATEMENT LOAD EXTENSION "${KUZU_ROOT_DIRECTORY}/extension/json/build/libjson.kuzu_extension";
+---- ok
+-STATEMENT CREATE NODE TABLE person(ID UINT64, PRIMARY KEY (ID))
+---- ok
+-STATEMENT CREATE REL TABLE knows(FROM person TO person)
+---- ok
+-STATEMENT COPY person FROM "${KUZU_ROOT_DIRECTORY}/dataset/json-error/person_ok.json";
+---- ok
+-STATEMENT COPY knows FROM "${KUZU_ROOT_DIRECTORY}/dataset/json-error/index_lookup_error.json"(IGNORE_ERRORS=true);
+---- ok
+-STATEMENT CALL show_warnings() RETURN message, file_path, line_or_record_number, skipped_line_or_record
+---- 2
+Unable to find primary key value 5.|${KUZU_ROOT_DIRECTORY}/dataset/json-error/index_lookup_error.json|3|{"from": 1, "to": 5}
```

Comment:
I am little bit confused about the term `record` in json. Should we simply show the line number in the original json file? let's discuss this.

Reply:
Discussed offline, we will show line number + skipped line/record